docs-ref-services/sql.md

```diff
@@ -19,46 +19,67 @@ ms.service:
 
 Work with data stored in  [Azure SQL Database](https://docs.microsoft.com/azure/sql-database/sql-database-technical-overview) from Python with the Microsoft ODBC driver. 
 
-The management libraries provide an interface to create, manage, and scale Azure SQL Database deployments from your Python code. Set up and manage databases in [elastic pools](https://docs.microsoft.com/en-us/azure/sql-database/sql-database-elastic-pool) to share resources and configure databases across multiple regions from your code.
+## Client ODBC driver and pyodbc
```

Comment:
Maybe change the caption to make this about prerequisites?
pyodbc must be installed